examples/research_projects/promptdiffusion/README.md

```diff
@@ -0,0 +1,42 @@
+### PromptDiffusion Pipeline
+
+From the project [page](https://zhendong-wang.github.io/prompt-diffusion.github.io/)
+
+"With a prompt consisting of a task-specific example pair of images and text guidance, and a new query image, Prompt Diffusion can comprehend the desired task and generate the corresponding output image on both seen (trained) and unseen (new) task types."
+
+For any usage questions, please refer to the [paper](https://arxiv.org/abs/2305.01115).
+
+
+```py
+# Prepare models by converting them from the checkpoint https://huggingface.co/zhendongw/prompt-diffusion
+import torch
+import diffusers
+from diffusers import UniPCMultistepScheduler
+from diffusers.utils import load_image
+from promptdiffusioncontrolnet import PromptDiffusionControlNetModel
+from diffusers.pipelines.pipeline_utils import DiffusionPipeline
+
+
+from PIL import Image,ImageOps
+
+image_a = ImageOps.invert(load_image("https://github.com/Zhendong-Wang/Prompt-Diffusion/blob/main/images_to_try/house_line.png?raw=true"))
+
+image_b = load_image("https://github.com/Zhendong-Wang/Prompt-Diffusion/blob/main/images_to_try/house.png?raw=true")
+query = ImageOps.invert(load_image("https://github.com/Zhendong-Wang/Prompt-Diffusion/blob/main/images_to_try/new_01.png?raw=true"))
+
+# load prompt diffusion control net and prompt diffusion
+
+controlnet = PromptDiffusionControlNetModel.from_pretrained("path-to-promptdiffusion-controlnet", torch_dtype=torch.float16)
```

Comment:
I think it makes sense to first talk about how this checkpoint was obtained i.e., an example command of running the conversion.

Reply:
Could you provide the checkpoint path from the Hub directly? I.e., it could make sense to directly load it from the Hub? A few options:

* Host it under your HF profile while clearly providing the citations to the original model. 
* Reach out to the author to see if they'd be willing to have the converted checkpoint under their HF profile. 

Let me know what you think.

Reply:
https://github.com/Zhendong-Wang/Prompt-Diffusion/issues/12 No official release date, so mine has to suffice.